### ISSUE TITLE
Bug when checking if `hasher_features` is a valid option

### ISSUE THREAD

**Issue by desilinguist** [EducationalTestingService/skll#234]
In `experiments.py`, we have the following code:

```python
# produce warnings if hasher_features is set but feature_hasher
# is missing or set to False.
if config.has_option("Input", "hasher_features"):
    if not config.has_option("Input", "hasher_features"):
        logger.warning("Ignoring hasher_features since feature_hasher" +
                       " is missing from the config file.")
    else:
        feature_hasher = config.getboolean("Input", "feature_hasher")
        if not feature_hasher:
            logger.warning("Ignoring hasher_features since feature_hasher" +
                           " is set to False.")
```

The second `if` statement should read:
```python
    if not config.has_option("Input", "feature_hasher"):
```

**Comment by dan-blanchard** [comment]
Man, who would have done such a thing? (7a434e2173a) :stuck_out_tongue_winking_eye:

**Comment by desilinguist** [comment]
I figured it was a copy-paste flub and I am the master of those :)

**Comment by dan-blanchard** [comment]
Yeah, and it doesn't help how similar those two settings names are.

**Comment by desilinguist** [comment]
Addressed by #245.
